packages/c/sshnpd/src/device_info.c

```diff
@@ -0,0 +1,166 @@
+#include "atlogger/atlogger.h"
+#include "sshnpd/daemon.h"
+#include "sshnpd/params.h"
+#include <atclient/atclient.h>
+#include <sshnpd/device_info.h>
+#include <string.h>
+#include <time.h>
+
+int handle_username_keys(atclient *atclient, const char **atsigns, size_t num_atsigns, const char *username,
+                         const char *device_name, const char *device_atsign, bool make_visible) {
```

Comment:
previously defined in background_jobs.c, this puts or deletes (during startup of sshnpd) all of the unix username keys, depending on whether the hide flag was passed to the program.